pallets/runtime/tests/src/identity_test.rs

```diff
@@ -1557,3 +1558,69 @@ fn add_permission_with_secondary_key() {
             assert_eq!(sig_items[1], sig_2);
         });
 }
+
+#[test]
+fn add_investor_uniqueness_claim() {
+    ExtBuilder::default()
+        .cdd_providers(vec![AccountKeyring::Charlie.public()])
+        .build()
+        .execute_with(|| do_add_investor_uniqueness_claim());
+}
+
+fn do_add_investor_uniqueness_claim() {
+    let alice = User::new(AccountKeyring::Alice);
+    let cdd_provider = AccountKeyring::Charlie.public();
+    let ticker = simple_asset(alice.acc(), true).1;
```

Comment:
Let's avoid adding more technical debt and use `basic_asset` instead.

Reply:
I don't get your point. To use `basic_asset` I would need to define a `SecurityToken` anyway which is what `simple_asset` is doing.

Reply:
`fn simple_asset` duplicates code and shouldn't exist. You're adding another dependency to it. Instead, use `a_token` to produce `(Ticker, SecurityToken<u128>)` to pass to `basic_asset`, or if you prefer, create `fn asset(owner: User) -> Ticker` as a wrapper for that logic.

Reply:
Done. Also marked `simple_asset` as deprecated.